crates/hc_sleuth/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "hc_sleuth"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0"
+aitia = { path = "../aitia" }
+derive_more = "0.99"
+holochain_state_types = { path = "../holochain_state_types" }
```

Comment:
Don't know what the release automation will do with this. Might be fine but I think it will write it back with the latest versions. Probably safest to include them with the path deps?

Reply:
Yeah, I'm slowly going through all of these little details as CI complains.